### PR TITLE
added support for dlc heads and removed runnerhead.as include in quarters.as

### PR DESCRIPTION
**Pull request by makmoud98**
this just simplifies the code. my main motive for this is to be able to add hooks like onInit and onCommand into runnerhead. before this change it was not possible because the compiler thinks there is already a function with the same name and parameter present (since runnerhead is included at the top). this change will allow me to update the playerheads mod without having to make changes to the quarters script. this helps to keep the playerheads mod to be as compatible with other mods as much as possible. thanks for reading

### CODE REVIEW

**Comment by fzzle** [comment]
Nice modification. I wanted to do this for some time. Implementation scripts (w/ hooks) shouldn't be used as common scripts (included). It also removes the head gib created for quarters  (`onDie` in RunnerHead.as).

File: Entities/Industry/CTFShops/Quarters/Quarters.as [hunk 1]
```diff
@@ -268,6 +266,7 @@ void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint@ attachedPoint)
 	attached.AddScript("WakeOnHit.as");
 
 	string texName = default_head_path;
```
Comment (fzzle) [inline Entities/Industry/CTFShops/Quarters/Quarters.as]: Compile error: `default_head_path` is still used.
Could set it to `'Heads.png'`.

File: Entities/Industry/CTFShops/Quarters/Quarters.as [hunk 2]
```diff
@@ -277,6 +276,7 @@ void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint@ attachedPoint)
 		if (head !is null)
 		{
 			texName = head.getFilename();
+			frame = head.getFrame();
```
Comment (fzzle) [inline Entities/Industry/CTFShops/Quarters/Quarters.as]: `head.getFrame();` returns the frame of the spritesheet currently used in the animation. If I idle the spritesheet frame is offset + 0. However, if I (for example) slash the moment I enter the bed the spritesheet frame is offset + 1. Then `anim.AddFrame(frame + 2)` adds the spritesheet frame at offset + 3 (a blank frame).

Replace line 279:
```
Animation@ animation = head.getAnimation('default');
if (animation !is null)
{
  frame = animation.getFrame(2);
}
```

Replace line 299:
```
anim.AddFrame(frame);
```

**Comment by 1bardesign** [comment]
Hey; I'd love to go through and merge/close the current batch of pull requests next Monday (ie in a week) - if you could get this ready to go with the required changes as a PR I'd be happy to merge it in 👍

**Comment by 1bardesign** [comment]
Closing this now; went with https://github.com/transhumandesign/kag-base/pull/13 instead

**Pull request closed without merging**